pkg/controllers/matrixone/matrixonecluster_controller.go

```diff
@@ -75,6 +78,9 @@ func (r *ClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 func (r *ClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&v1alpha1.MatrixoneCluster{}).
+		Owns(&corev1.Service{}).
+		Owns(&appsv1.StatefulSet{}).
+		Owns(&policy.PodDisruptionBudget{}).
```

Comment:
Should we watch PDB now? Looks like cluster controller does not create any

Reply:
yeah, I removed it. adress in https://github.com/matrixorigin/matrixone-operator/pull/116/commits/5f18ab4a2d05dc764c4a3b60762da33d3b584128